www/include/configuration/configObject/servicegroup/getServiceXml.php

```diff
@@ -64,7 +64,7 @@
     		  WHERE s.service_id = hsr.service_service_id
     		  AND hsr.host_host_id = h.host_id ";
     if ($hostId) {
-        $query .= " AND h.host_id = " . $db->escape($hostId);
+        $query .= " AND h.host_id = " . (int)$hostId;
```

Comment:
you could avoid sending a useless query by checking if hostId is false on line 56 ->
https://github.com/centreon/centreon/pull/8758/files#diff-d4cc75f9c7c6ad77d67f6868d48c2574R56